CMS-Enterprise/mint-app
# [MINT-3227] API Support for deleting an MTO Milestone

# MINT-3227

## Description

- Adds API Support (GraphQL mutation) for deleting an MTO milestone
- Adds Postman Query
- Adds resolver test (honestly pretty minimal at this point, maybe could bolster this up)

> [!NOTE]
> This PR also introduces a `sqlutils.ExecProcedure` method, useful for calling delete statements which do not need to return any values

## How to test this change

1. Use `scripts/dev db:seed` to populate an MTO with some milestones
2. Optionally create some milestones from the common milestone library and link some solutions
3. Use the postman query to delete those milestones
  a. Ensure that the milestones are deleted properly
  b. Ensure that any milestone solution links are also deleted (but _not_ the solutions themselves

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
